core/src/main/java/org/frankframework/soap/SoapWrapperPipe.java

```diff
@@ -50,6 +50,7 @@ public class SoapWrapperPipe extends FixedForwardPipe implements IWrapperPipe {
 	protected static final String DEFAULT_SOAP_HEADER_SESSION_KEY = "soapHeader";
 	protected static final String DEFAULT_SOAP_NAMESPACE_SESSION_KEY = "soapNamespace";
 	protected static final SoapVersion DEFAULT_SOAP_VERSION_FOR_WRAPPING = SoapVersion.SOAP11;
+	public static final String DEFAULT_XML_HEADER = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";
```

Comment:
Ik denk niet dat we de character encoding moeten hard-coden in onze output. Dan kunnen we nog wel eens voor nare verassingen komen te staan, denk ik.

Reply:
Ik heb het nu aangepast, zodat hij de encoding uit het Message-object haalt, welke hij gebruikt.